README.md

```diff
@@ -1,13 +1,11 @@
 # Survey form with React
 
-Replace this readme with your own information about your project.
-
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+In this project I used React to build a survey form. When the user have answered all the questions and submitted the form a summary of the answers is displayed. 
 
 ## The problem
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+I started out by creating a component for each question. Later on I changed the React flow since I didn't know how to send the values from my functions back to the App.js file. 
```

Comment:
You have TaskRadioOption and ReasonRadioOption components, could sending the values be done somehow as you did with those components? I don't know, I was just wondering this because I want to understand how to structure React apps.😄